tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -1,134 +0,0 @@
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-# SPDX-License-Identifier: Apache-2.0
-import pytest
-
-from constants import TRUST_STORE_BUNDLE, TRUST_STORE_TRUSTED_BUNDLE
-from configuration import PROTOCOLS
-from common import ProviderOptions, Ciphers, pq_enabled
-from fixtures import managed_process  # lgtm [py/unused-import]
-from global_flags import get_flag, is_criterion_on, S2N_FIPS_MODE, S2N_USE_CRITERION
-from providers import Provider, S2N
-from test_pq_handshake import PQ_ENABLED_FLAG
-from utils import invalid_test_parameters, get_parameter_name, to_bytes
-
-
-ENDPOINTS = [
```

Comment:
To confirm: all of these endpoints are represented in your new tests? It's hard to visually diff-- have you confirmed somehow?

Reply:
Yes, we can just use a line count as a sanity check.

The existing test has 23 + 1 (akamai) lines. (not including all of the very old commented out ones, which were removed).

The new test has
- http: 23
- tls: 2 (akamai, kms)

The additional case is from adding `cloudfront`.